guides/common/modules/snip_red-hat-helper-apps.adoc

```diff
@@ -0,0 +1,11 @@
+ifdef::upgrading-connected,upgrading-disconnected[]
+For interactive upgrade instructions, you can also use the {ProjectName} Upgrade Helper on the Red{nbsp}Hat Customer Portal.
```

Comment:
Drop `also` here and on line 8?  This implies there is multiple interactive methods.

Reply:
Good catch, updated.